packages/solana/lib/src/crypto/ed25519_hd_keypair.dart

```diff
@@ -135,8 +135,18 @@ class Ed25519HDKeyPair extends KeyPair {
   }
 
   /// Build a derivation path with [account] and [change]
-  static String _getHDPath(int account, int change) =>
-      "m/44'/501'/$account'/$change'";
+  static String _getHDPath(int? account, int? change) {
```

Comment:
I think we do a similar stuff on the app side [in this file](https://github.com/espresso-cash/espresso-cash-public/blob/0276db07afead3a34a98efbe8aa021f59f4efc5e/packages/espressocash_app/lib/core/wallet.dart#L30) regarding Saga wallets. Should we update there as well?

Reply:
That would be a breaking change for the app users, not sure if we should do it.

Also, in the app, we should actually provide `/0/0` when we call `fromMnemonic`, otherwise, again, it would be a breaking change for the app users.

Reply:
Yes, for the app we have already set `/0/0` for local wallets